.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,9 +19,35 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 ## Proof that changes are correct
 
 <!--
-Add videos/screenshots of the user-facing interface in various display sizes (mainly phone, tablet, and desktop display size) to demonstrate that the changes made in this PR work correctly. Please also include videos/screenshots of the developer tools browser console, so that we can be sure that there are no console errors.
-If the PR changes the UI in any of the files listed in rtl_css.py (i.e, those that have a separate .rtl.css file for styling), make sure to add screenshots with the site language set to Arabic as well.
-If the changes in your PRs are autogenerated via a script and you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}".
+Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR work correctly. Please also include videos/screenshots of the developer tools browser console, so that we can be sure that there are no console errors.
+If the changes in your PRs are autogenerated via a script and you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}" and remove all the sections below.
+-->
+
+#### Proof of changes on desktop
+
+<!--
+Make sure to properly verify that everything works correctly. That there 
+are not weird UI mistakes and other problems. Also if there are any newly added
+fiels try to fill them up and test out that different inputs are accepted.
+-->
+
+#### Proof of changes on mobile phone
+
+#### Proof of changes on slow/throttled network
+
+<!--
+Throttle the network (to 3G) using the browser Developer Tools and then provide 
+a video similar to the one you added above. There should be no performance or 
+UI issues in while the network is slow. This video can be skipped if the changes 
+are not frontend related.
+-->
+
+#### Proof of changes in Arabic language
+
+<!--
+If the PR changes the UI in any of the files listed in rtl_css.py (i.e, those that have 
+a separate .rtl.css file for styling), make sure to add screenshots with the site 
+language set to Arabic as well. 
```

Comment:
Maybe worth explaining why we chose Arabic here so that developers understand the rationale

Reply:
Done.